chapter_multilayer-perceptrons/underfit-overfit.md

```diff
@@ -419,7 +419,7 @@ true_w = np.zeros(maxdegree)  # Allocate lots of empty space
 true_w[0:4] = np.array([5, 1.2, -3.4, 5.6])
 
 features = np.random.normal(size=(n_train + n_test, 1))
-features = np.random.shuffle(features)
```

Comment:
`features` should be assigned by the shuffled result. Here shuffle is not an in-place operation